### PR TITLE
Update Batch Size to Eliminate SQL Error SQLITE_ERROR: too many SQL variables

### PR DESCRIPTION
**Pull request by jbelke**
Update Batch Size to Eliminate SQL Error SQLITE_ERROR: too many SQL variables

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -12,8 +12,11 @@ updates:
       - dependencies
     rebase-strategy: disabled
     ignore:
-      - dependency-name: "eslint*"
-      - dependency-name: "prettier*"
+      - dependency-name: "*eslint*"
+      - dependency-name: "*jest*"
+      - dependency-name: "*cypress*"
+      - dependency-name: "*prettier*"
+      - dependency-name: "*concurrently*"
```
Comment (bidoubiwa) [inline .github/dependabot.yml:19]:
```suggestion
      - dependency-name: "*eslint*"
      - dependency-name: "*prettier*"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -26,4 +29,7 @@ updates:
       - dependencies
     rebase-strategy: disabled
     ignore:
-      - dependency-name: "eslint*"
+      - dependency-name: "*eslint*"
+      - dependency-name: "*jest*"
+      - dependency-name: "*cypress*"
+      - dependency-name: "*prettier*"
```
Comment (bidoubiwa) [inline .github/dependabot.yml:35]:
```suggestion
      - dependency-name: "*eslint*"
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

**Comment by bidoubiwa** [comment]
Dependabot file is still present in the files changed, could you remove the file from the PR?

**Review by jbelke — COMMENTED** [review]
Approved

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥 Thnks a lot

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**